Gemfile

```diff
@@ -15,7 +15,7 @@ gem 'ya2yaml'
 gem 'i18n'
 gem 'dynamic_form'
 gem 'exception_notification'
-gem 'activerecord-import'
+gem 'activerecord-import', '>= 0.2.0'
```

Comment:
Do you think this is needed? The wiki said I should be using the latest version, however upon doing a gem upgrade... I don't recall it upgrading and it appeared to have used 0.10.0
Probably minor since this is working well regardless of the version currently.